src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -76,7 +79,11 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
   } = services;
   const addBasePath = http.basePath.prepend;
   const currentTheme = useObservable(theme.theme$, { darkMode: false, name: 'amsterdam' });
-
+  const { euiTheme } = useEuiTheme();
+  const minBreakpointM = useEuiMinBreakpoint('m');
+  const className = classNames(
+    !newsFetchResult?.feedItems?.length && 'kbnOverviewSupplements--noNews'
+  );
```

Comment:
we might rewrite this like so; 

```suggestion
classnames({
'kbnOverviewSupplements--noNews': !newsFetchResult?.feedItems?.length
})

```